### DISCUSSION TITLE
Any issue that I can help with before applying for GSOC 2023?

### DISCUSSION THREAD

**Discussion by SanftMonster** [openvinotoolkit/openvino#15741] [Google Summer of Code]
Dear OpenVINO developers,

I’m a student with a strong interest in deep learning and AI framework and I'd like to apply for GSOC 2023 of OpenVINO community. Before my application of GSOC, I’d like to participate in some small issues of OpenVINO to gain some experience. However I noticed that issues of OpenVINO are always assigned to internal developers soon. If there are any small issues or tasks that I can assist with, I would be more than happy to help. I am excited about the opportunity to contribute to this community.

I tried to ask for this question via emailing openvino-gsoc@lists.linuxfoundation.org a week ago but got no reply. Hope that this discussion does not disturb you much.

Thank you for your time and consideration.

Best regards,
Rinne

**Comment by mlukasze** [comment]
hey @AsakusaRinne 
your timing is perfect!

some of our developers will back to you soon, please stay tune!

**Reply by SanftMonster** [reply]
OK, Thanks! 😁

**Comment by AnshTanwar** [comment]
Hi, I am also looking for a similar opportunity.
